src/main/groovy/jenkins/automation/builders/JsJobBuilder.groovy

```diff
@@ -4,14 +4,35 @@ import javaposse.jobdsl.dsl.DslFactory
 import javaposse.jobdsl.dsl.Job
 import jenkins.automation.utils.ScmUtils
 
+
+/**
+ * @param name          Job name
+ * @param description   Job description
+ * @param pollSchedule  Optional
+ * @param artifacts     Optional Closure defaulted to closure below :
+                                  {
+                                     pattern("dist/")
+                                     fingerprint()
+                                     defaultExcludes()
+
+                                     }
+
+ * @param emails  List or String of notification email addresses
+ * @param repos List of repos to watch
+ * @param use_versions same as BaseJobBuilder
```

Comment:
Sure. The documentation can have a bit more meat on it. But checkout this: http://cfpb.github.io/jenkins-automation/.